WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -62,6 +63,11 @@ class PostListItemViewHolder(
         }
     }
 
+    private fun updateStatusLabels(view: WPTextView, statusLabels: List<UiString>, delimiter: UiString) {
+        val separator = uiHelpers.getTextOfUiString(view.context, delimiter)
```

Comment:
Oh, no! While `joinToString` is correct, this now changed the behavior to join the strings without a space between them. So the text will now be `"Local changes•Private"` instead of `"Local changes • Private"`. Let me know if you'd like to fix that on a separate PR @malinajirka. Sorry bout that!

Reply:
No worries at all. I feel ashamed I apparently haven't tested it after the changes:X. I'm still not sure where they got lost, but fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9491/commits/cc8b14066dccd136d461b5afc4fc0573e16c57ab.

Reply:
TIL Android trims the spaces in `strings.xml`. 😄